triehash/benches/triehash.rs

```diff
@@ -65,7 +67,7 @@ fn triehash_insertions_32_mir_1k(b: &mut Bencher) {
 	};
 	let d = st.make();
 	b.iter(&mut ||{
-		trie_root(d.clone()).clone();
+		let _ = trie_root::<KeccakHasher, _, _, _>(d.clone()).clone();
```

Comment:
Should we use [`black_box`](https://doc.rust-lang.org/1.7.0/book/benchmark-tests.html#gotcha-optimizations) to ensure it is not optimized away?

Reply:
Yeah, I was considering doing that. I *think* `iter()` uses it [internally though](https://docs.rs/bencher/0.1.5/src/bencher/lib.rs.html#607) but tbh I am not sure what `cargo bench` actually ends up running or if that code is old code (and if so, what replaced it?).

I also found some docs claiming `black_box` is useful when the optimizer throws away e.g. an iterator passed as an argument but that's not the case here. So I just went for something to shut up the linter.